apps/desktop/src/routes/+page.svelte

```diff
@@ -39,13 +40,13 @@
 		{ type: 'loading' } as Redirect
 	);
 
-	$: {
+	run(() => {
 		if ($redirect.type === 'redirect') {
 			goto($redirect.subject);
 		} else if ($redirect.type === 'no-projects') {
 			goto('/onboarding');
 		}
-	}
+	});
```

Comment:
Should we be fixing these `run`s?

Reply:
Yeah slowly but surely going through and removing all the svelte/legacy imports 👍